app/models/hubstats/pull_request.rb

```diff
@@ -130,5 +142,27 @@ def self.group_by(group)
         scoped
       end
     end
+
+    # Public - Assigns a team to a pull request based on the user's team
+    #
+    # Returns - nothing
+    def assign_team_from_user
+      puts self.inspect
+      user = Hubstats::User.where(id: self.user_id).first
```

Comment:
You can use find for primary key.
http://guides.rubyonrails.org/v3.2.21/active_record_querying.html#retrieving-a-single-object